src/sheet/SheetEngineAdapter.js

```diff
@@ -181,11 +181,9 @@ class CellAdapter {
       this._lang = null
       let sheet = cell.getDocument()
       let type = sheet.getCellType(cell)
+      let text = cell.text()
       // HACK: this marshalling should be done somewhere else
```

Comment:
Seems that you fixed this HACK

Reply:
If so, you could remove this comment

Reply:
I'm not sure. This was quite a quick fix and still feels temporary. I'm going to propose a change in how we deal with types, including casting, so we may deal with this comment then.

Reply:
ok